src/portable/renesas/usba/dcd_usba.c

```diff
@@ -689,6 +695,17 @@ void dcd_int_handler(uint8_t rhport)
       dcd_disconnect(rhport);
     }
   }
+  if (is0 & USB_IS0_RESM) {
+    dcd_event_bus_signal(rhport, DCD_EVENT_RESUME, true);
+    _dcd.suspended = 0;
+  }
+  if (is0 & USB_IS0_SOFR) {
+    if (_dcd.suspended) {
+      dcd_event_bus_signal(rhport, DCD_EVENT_RESUME, true);
```

Comment:
hmm, this is a bit odd, SOF should be recognized by hardware as Resume signal. Do we really need to do it manual ?

Reply:
I am agree your guessing. But unfortunately, RX does not signal resume interrupt caused by `dcd_remote_wakeup()` , in my experience. Of course, when the host resumes on its own, it occurs resume interrupt.

Reply:
@kkitayam I see, that is a bit odd for rx65n controller for not raising Resume interrupt in this case. Could you put an comment explaining why we need to manually send resume event here. Just in case when going back to this and/or we could figure out a better way to detect resume.

Reply:
Comments are added at 1ff3b76.